Sources/NIOWebSocket/WebSocketOpcode.swift

```diff
@@ -57,7 +57,24 @@ extension WebSocketOpcode: Equatable {}
 extension WebSocketOpcode: Hashable {}
 
 extension WebSocketOpcode: CaseIterable {
-    public static var allCases = (0..<0x10).map { WebSocketOpcode(rawValue: $0) }
+    public static let allCases: [WebSocketOpcode] = [
```

Comment:
So this has the unfortunate property of being a technical API break.

Can I suggest making this a computed property that returns the array on get, and on set just does nothing?

Reply:
👍 sure. I'd misread this as already being a get-only computed property so I missed the break. I've also deprecated the setter.